zulipterminal/model.py

```diff
@@ -1012,25 +1012,31 @@ def _handle_update_message_flags_event(self, event: Event) -> None:
         """
         Handle change to message flags (eg. starred, read)
         """
+        if (self.server_feature_level is None
+                or self.server_feature_level < 32):
+            operation_key = 'operation'  # type: Literal['operation', 'op']
+        else:
+            operation_key = 'op'
```

Comment:
It might cleaner to have this set `operation = event['operation']` or `operation = event['op']` and remove the `operation_key` variable.